src/dataset.ts

```diff
@@ -75,8 +75,14 @@ export class OdrDatasets extends Stack {
           {
             // Ensure files are in intelligent tiering access
             transitions: [{ storageClass: StorageClass.INTELLIGENT_TIERING, transitionAfter: Duration.days(0) }],
-            // Delete any old versions of files after 30 days
-            noncurrentVersionExpiration: Duration.days(30),
+            // Retain noncurrent versions in Standard for 30 days, then move to Deep Archive
```

Comment:
If I understand this transition rule, it will retain the 100 most recent noncurrent versions in the current storage class and only versions older than the 100 most recent noncurrent versions AND older than 30 days will be transitioned. 
Is that correct?

Reply:
You may want to update the PR description with a new diff?